src/main/java/org/togetherjava/discordbot/commands/parsers/UserParsers.java

```diff
@@ -13,6 +13,10 @@
  */
 public class UserParsers {
 
+  private UserParsers() {
+    throw new IllegalStateException("Utility class");
+  }
```

Comment:
It might be important to decide if we should throw an exception like this to state that this is a utility class or just have a single line of `private UserParsers() {}` and expect future contributors of this class or any other util class to not instantiate this class? Since we are targeting this repository towards beginners, it might make sense. However, this may not be necessary.